packages/dev/serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -78,6 +78,8 @@ function getFileExtensionFromMimeType(mimeType: ImageMimeType): string {
             return ".png";
         case ImageMimeType.WEBP:
             return ".webp";
+        case ImageMimeType.AVIF:
+            return ".avif";

```

Comment:
Probably should update this also?
https://github.com/BabylonJS/Babylon.js/blob/ad4785d377ffe5d4ad9b4dc58a31ae74a2951825/packages/dev/serializers/src/glTF/2.0/glTFMaterialExporter.ts#L1089-L1102